vue-training/src/components/TechBox.vue

```diff
@@ -1,11 +1,13 @@
 <template lang="pug">
   .tech-box
+    //- TODO: resolve error when I add this :alt='tech.title'
```

Comment:
Why are you having this issue?

Reply:
It works but shows this error when compiling 
<img width="1680" alt="Screen Shot 2019-04-29 at 11 01 10 AM" src="https://user-images.githubusercontent.com/822241/56911762-5b79c100-6a73-11e9-8616-ba5e15dc5988.png">

Reply:
missing ,?

Reply:
yes, that was solved